app/code/Magento/Inventory/Setup/InstallSchema.php

```diff
@@ -53,13 +59,15 @@ public function __construct(
         CreateSourceCarrierLinkTable $createSourceCarrierLinkTable,
         CreateSourceItemTable $createSourceItemTable,
         CreateStockTable $createStockTable,
-        CreateStockSourceLinkTable $createStockSourceLinkTable
+        CreateStockSourceLinkTable $createStockSourceLinkTable,
+        CreateStockItemIndexTable $createStockItemIndexTable
```

Comment:
No need to introduce index table during the installation , it should be done at the time of re-indexation